content/blog/2016/2016-09-01-jenkins-world-contributor-summit.adoc

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: "Jenkins World Contributor Summit"
+tags:
+- event
+- JenkinsWorld
+author: rtyler
+---
+
+
+image:/images/conferences/Jenkins-World_125x125.png[Jenkins World, role=right]
+
+
+At previous Jenkins User Conferences we have hosted "Contributor Summits" to
+gather developers and power-users in one room to discuss specific areas of
+Jenkins, such as Scalability, Pipeline, etc. At this year's
+link:https://www.cloudbees.com/jenkinsworld/home[Jenkins World] we're planning
```

Comment:
Something like: 
"At this year's JW +Contributor Summit+, we're planning"

Or 
"We'll be doing this again at this year's JW, and we're planning"